spec.html

```diff
@@ -10697,6 +10704,7 @@ <h2>Syntax</h2>
       SingleLineCommentChar ::
         SourceCharacter but not LineTerminator
     </emu-grammar>
+    <p>The semantics and syntax are extended in <emu-xref href="#sec-html-like-comments"></emu-xref></p>
```

Comment:
I'd prefer the same wording I suggested above here as well:

```suggestion
    <p>A number of productions in this section are given alternative definitions in section <emu-xref href="#sec-html-like-comments"></emu-xref></p>
```